src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom.py

```diff
@@ -0,0 +1,349 @@
+    protection_containers_cf,
+    backup_operation_statuses_cf,
+    protection_container_refresh_operation_results_cf)
+
+logger = azlogging.get_az_logger(__name__)
+
+def show_policy(client, policy_name, vault):
+    protection_policies_client = protection_policies_cf()
+
+    rs_vault = _get_vault_from_json(client, vault)
+    resource_group = _get_resource_group_from_id(rs_vault.id)
+
+    policy = protection_policies_client.get(rs_vault.name, resource_group, policy_name)
+    return policy
+
+def list_policies(client, vault):
+    backup_policies_client = backup_policies_cf()
+
+    rs_vault = _get_vault_from_json(client, vault)
+    resource_group = _get_resource_group_from_id(rs_vault.id)
+
+    policies = backup_policies_client.list(rs_vault.name, resource_group)
+    return _get_one_or_many(_get_paged_list(policies))
+
+def show_container(client, container_name, vault, container_type="AzureVM", status="Registered"):
+    backup_protection_containers_client = backup_protection_containers_cf()
+
+    rs_vault = _get_vault_from_json(client, vault)
+    resource_group = _get_resource_group_from_id(rs_vault.id)
+
+    backup_management_type = _get_backup_management_type(container_type)
+
+    filter_string = _get_filter_string({
+        'friendlyName' : container_name,
+        'backupManagementType' : backup_management_type,
+        'status' : status})
+
+    containers = backup_protection_containers_client.list(rs_vault.name, resource_group, filter_string)
+    return _get_one_or_many(_get_paged_list(containers))
+
+def list_containers(client, vault, container_type="AzureVM", status="Registered"):
+    backup_protection_containers_client = backup_protection_containers_cf()
+
+    rs_vault = _get_vault_from_json(client, vault)
+    resource_group = _get_resource_group_from_id(rs_vault.id)
+
+    backup_management_type = _get_backup_management_type(container_type)
+
+    filter_string = _get_filter_string({
+        'backupManagementType' : backup_management_type,
+        'status' : status})
+
+    containers = backup_protection_containers_client.list(rs_vault.name, resource_group, filter_string)
+    return _get_one_or_many(_get_paged_list(containers))
+
+def enable_protection_for_vm(client, vm, vault, policy):
+    # Client factories
+    protection_policies_client = protection_policies_cf()
+    protected_items_client = protected_items_cf()
+
+    # Get objects from JSON files
+    vm = _get_vm_from_json(virtual_machines_cf(), vm)
+    rs_vault = _get_vault_from_json(client, vault)
+    resource_group = _get_resource_group_from_id(rs_vault.id)
+    policy = _get_policy_from_json(protection_policies_client, policy)
+
+    # VM name and resource group name
+    vm_name = vm.name
+    vm_rg = _get_resource_group_from_id(vm.id)
+
+    # Get protectable item.
+    protectable_item = _get_protectable_item(rs_vault.name, resource_group, vm_name, vm_rg)
+    if protectable_item is None:
+        raise CliError("""
```

Comment:
Have you imported this? Doesn't look like this would work. I could tell because at least it should be `CLIError` unless it's being renamed somewhere else.